ras/lshw.py

```diff
@@ -123,9 +126,9 @@ def test_lshw_verification(self):
             self.fail("lshw failed to show correct mac address")
 
         # verify network
-        if self.active_interface\
-                not in self.run_cmd_out("lshw -class network"):
-            self.fail("lshw failed to show correct active network interface")
+        if distro.detect().arch not in ['ppc64le', 'ppc64el']:
```

Comment:
this make test not run ppc64le system so not require so earlier test code is fine